src/tartiflette_asgi/_endpoints.py

```diff
@@ -61,12 +61,13 @@ async def _get_response(self, request: Request, data: QueryParams) -> Response:
 
         config = get_graphql_config(request)
         background = BackgroundTasks()
-        context = {"req": request, "background": background, **config.context}
+        context_additions = {"req": request, "background": background}
+        config.context.update(context_additions)
```

Comment:
Would a shallow copy be more suitable, so that we don't pollute the initial context with these request-level values?

```suggestion
        context = config.context.copy()
        context.update({"req": request, "background": background})
```

Reply:
I agree with copying, but would it be okay if we use copy.copy instead of dict.copy, as that would preserve the initial object? In the case of subclassing something like UserDict, you would get a vanilla dict out of dict.copy instead of the custom class but you would get the custom class out of copy.copy.